Dockerfile

```diff
@@ -72,68 +71,131 @@ FROM builder-base AS pcov-builder
 RUN pecl install pcov && docker-php-ext-enable pcov
 
 
-# Stage that builds all other extensions
-FROM builder-base AS extensions-builder
-RUN docker-php-ext-install bcmath intl zip pcntl bz2
+# Stage that builds bcmath
+FROM builder-base AS bcmath-builder
+RUN docker-php-ext-install bcmath
+
+
+# Stage that builds intl
+FROM builder-base AS intl-builder
+RUN docker-php-ext-install intl
+
+
+# Stage that builds zip
+FROM builder-base AS zip-builder
+RUN docker-php-ext-install zip
+
+
+# Stage that builds pcntl
+FROM builder-base AS pcntl-builder
+RUN docker-php-ext-install pcntl
+
+
+# Stage that builds bz2
+FROM builder-base AS bz2-builder
+RUN docker-php-ext-install bz2
+
+
+# Stage with PHP extensions, all validated
+FROM php-base AS all-extensions
+
+COPY --link --from=ast-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=ast-builder /usr/local/etc/php/conf.d /usr/local/etc/php/conf.d
+COPY --link --from=pcov-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=pcov-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=bcmath-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=bcmath-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=intl-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=intl-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=zip-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=zip-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+COPY --link --from=bz2-builder /usr/local/lib/php/extensions /usr/local/lib/php/extensions
+COPY --link --from=bz2-builder /usr/local/etc/php/conf.d  /usr/local/etc/php/conf.d
+
+# Validate the extension configuration
+RUN php -m | grep ast
+RUN php -m | grep pcov
+RUN php -m | grep bcmath
+RUN php -m | grep intl
+RUN php -m | grep zip
+RUN php -m | grep bz2
```

Comment:
```suggestion
RUN set -eux; \
        php --re ast;
        php --re pcov;
        php --re bcmath;
        php --re intl;
        php --re zip;
        php --re bz2;

```

Reply:
Thanks; I'll leave as multiple layers 'cause it's not longer in the final image

Reply:
👍